Form/Type/DynamicFormType.php

```diff
@@ -149,6 +157,21 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             'mapped' => false,
         ]);
 
+        if ($this->honeyPotField) {
+            $builder->add(
+                str_replace(' ', '_', strtolower($this->honeyPotField)),
+                EmailType::class,
+                [
+                    'label' => $this->honeyPotField,
+                    'mapped' => false,
+                    'attr' => [
+                        'honeypot' => true
+                    ],
```

Comment:
attribute should not be here.

Instead it should be:

```php
'block_name' => 'honeypot',
```

See https://symfony.com/doc/3.4/reference/forms/types/form.html#block-name

The block_name should as in the docs be then `_honeypot_row`.